### DISCUSSION TITLE
TrustWalletCore library not found.

### DISCUSSION THREAD

**Discussion by nexiz578** [trustwallet/wallet-core#3272] [Q&A]
When compiling the cpp example, I get an error:
~/wallet-core/samples/cpp# cmake . -DWALLET_CORE=../../
CMake Error at CMakeLists.txt:34 (message):
  TrustWalletCore library not found.  Please provide TrustWalletCore build
  directory with -DWALLET_CORE.  Example: cmake .
  -DWALLET_CORE=../wallet-core

-- Configuring incomplete, errors occurred!
See also "/root/wallet-core/samples/cpp/CMakeFiles/CMakeOutput.log".

**Comment by Supiannpor** [comment]
> Saat mengkompilasi contoh cpp, saya mendapatkan kesalahan:~/wallet-core/samples/cpp# cmake . -DWALLET_CORE=.. /.. /Kesalahan CMake di CMakeLists.txt:34 (pesan):Perpustakaan TrustWalletCore tidak ditemukan. Tolong sediakan TrustWalletCore builddirektori dengan -DWALLET_CORE. Contoh: cmake .-DWALLET_CORE=../inti-dompet
> 
> -- Mengkonfigurasi tidak lengkap, kesalahan terjadi!Lihat juga "/root/wallet-core/samples/cpp/CMakeFiles/CMakeOutput.log".
